### PR TITLE
feat(NFT) add mint addresses

### PR DESCRIPTION
**Pull request by ashish4x**
adding the code to find the NFT mint addresses generated given Candy Machine public address.

### CODE REVIEW

File: code/nfts/nfts-mint-addresses/mint-addresses.en.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { Connection } from '@metaplex/js';
+import { programs } from '@metaplex/js';
+const { metadata: { Metadata, MetadataProgram } } = programs
```
Comment (jacobcreech) [inline code/nfts/nfts-mint-addresses/mint-addresses.en.ts:3]: In the latest build of `@metaplex/js`, these got moved out to `@metaplex-foundation/mpl-token-metadata`.

The line should be 
```javascript
import { Metadata, MetadataProgram } from '@metaplex-foundation/mpl-token-metadata';
```
instead

**Review by jacobcreech — CHANGES_REQUESTED** [review]
Awesome work! Just a small change requested and we'll get this merged in.

**Comment by ashish4x** [comment]
I've made the required changes. Please review.

**Review by jacobcreech — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**